opencti-platform/opencti-graphql/src/manager/retentionManager.ts

```diff
@@ -42,7 +45,7 @@ export const getElementsToDelete = async (context: AuthContext, scope: string, b
   if (scope === 'knowledge') {
     const jsonFilters = filters ? JSON.parse(filters) : null;
     const queryOptions = await convertFiltersToQueryOptions(jsonFilters, { before });
-    result = await elPaginate(context, RETENTION_MANAGER_USER, READ_STIX_INDICES, { ...queryOptions, first: RETENTION_BATCH_SIZE });
+    result = await elPaginate(context, RETENTION_MANAGER_USER, READ_ENTITIES_INDICES_WITHOUT_INFERRED, { ...queryOptions, first: RETENTION_BATCH_SIZE });
```

Comment:
why this change?

Reply:
Nice catch, it was a test. Will rollback

Reply:
done